### PR TITLE
[1LP][RFR] Fix the test for OSP

### PR DESCRIPTION
**Pull request by mnadeem92**
Signed-off-by: mnadeem92 <mnadeem@redhat.com>

{{ pytest: cfme/tests/v2v/test_v2v_migrations_single_vcenter.py -k "test_single_vm_migration_power_state_tags_retirement"  --use-provider osp13-ims  --use-provider vsphere67-ims --provider-limit 2 }}

### CODE REVIEW

File: cfme/tests/v2v/test_v2v_migrations_single_vcenter.py [hunk 1]
```diff
@@ -83,11 +88,12 @@ def test_single_vm_migration_power_state_tags_retirement(appliance, provider,
     assert migration_plan.wait_for_state("Successful")
 
     # check power state on migrated VM
-    rhv_prov = provider
-    migrated_vm = rhv_prov.mgmt.get_vm(src_vm.name)
-    assert power_state in migrated_vm.state
+    migrated_vm = get_migrated_vm(src_vm, provider)
+    vm_state = migrated_vm.state if provider.one_of(RHEVMProvider) else migrated_vm.mgmt.state
```
Comment (valaparthvi) [inline cfme/tests/v2v/test_v2v_migrations_single_vcenter.py:92]: Can't you just take `migrated_vm.mgmt.state` for both the providers? It should give you the same value.

File: cfme/tests/v2v/test_v2v_migrations_single_vcenter.py [hunk 2]
```diff
@@ -83,11 +88,12 @@ def test_single_vm_migration_power_state_tags_retirement(appliance, provider,
...
+    assert power_state in vm_state
```
Comment (valaparthvi) [inline cfme/tests/v2v/test_v2v_migrations_single_vcenter.py:93]: Shouldn't this be `assert power_state == vm_state`?

Comment (mnadeem92) [reply cfme/tests/v2v/test_v2v_migrations_single_vcenter.py:93]: It wont return the exact string, the out put is like "'VmState.RUNNING'"

File: cfme/tests/v2v/test_v2v_migrations_single_vcenter.py (hunk 1 above)
Comment (mnadeem92) [reply cfme/tests/v2v/test_v2v_migrations_single_vcenter.py:92]: Nice catch :)

**Pull request merged**